server/handle_map_layer_zxy.go

```diff
@@ -8,6 +8,8 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/go-spatial/geom"
+	"github.com/go-spatial/proj"
```

Comment:
I'm using `go-spatial/proj` because it's pure Go and it allows converting an arbitrary number of points in one pass. I considered `github.com/wroge/wgs84` which is also pure Go and better maintained, but it's a little more annoying to use.